<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imagegrabber tester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Server:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEditServerIp">
      <property name="text">
       <string>192.168.1.253</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QSpinBox" name="spinBoxServerPort">
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="value">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLabel" name="labelSocketStatus">
      <property name="text">
       <string>Not connected</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="labelBlink">
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QCheckBox" name="checkBoxAutoConnect">
      <property name="text">
       <string>Autoconnect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxGrabberTypes"/>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="pushButtonSettings">
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="pushButtonStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelFPSText">
      <property name="text">
       <string>FPS:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelFPSAvgText">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="labelFPSValue">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="labelFPSAvgValue">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;FPS&lt;span style=&quot; vertical-align:sub;&quot;&gt;avg&lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="10">
     <widget class="ImageView" name="graphicsViewImage"/>
    </item>
    <item row="0" column="8" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDiagrams">
    <property name="title">
     <string>Diagrams</string>
    </property>
    <addaction name="actionFPSDiagram"/>
   </widget>
   <widget class="QMenu" name="menuDelay_measurement">
    <property name="title">
     <string>Benchmarks</string>
    </property>
    <widget class="QMenu" name="menuLagg_measurements">
     <property name="title">
      <string>Lagg measurements</string>
     </property>
     <addaction name="actionTest_red"/>
     <addaction name="actionTest_blue"/>
     <addaction name="actionMeasure_delay"/>
     <addaction name="actionSelect_measurement_point"/>
    </widget>
    <widget class="QMenu" name="menuCompare_streams">
     <property name="title">
      <string>Compare streams</string>
     </property>
     <addaction name="actionCompareSettings"/>
    </widget>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="menuLagg_measurements"/>
    <addaction name="menuCompare_streams"/>
   </widget>
   <addaction name="menuDiagrams"/>
   <addaction name="menuDelay_measurement"/>
  </widget>
  <action name="actionFPSDiagram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
  </action>
  <action name="actionSelect_measurement_point">
   <property name="text">
    <string>Select measurement point</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionTest_red">
   <property name="text">
    <string>Test red</string>
   </property>
  </action>
  <action name="actionTest_blue">
   <property name="text">
    <string>Test blue</string>
   </property>
  </action>
  <action name="actionMeasure_delay">
   <property name="text">
    <string>Measure delay</string>
   </property>
  </action>
  <action name="actioCompareSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionStart_compare">
   <property name="text">
    <string>Start compare</string>
   </property>
  </action>
  <action name="actionCompareSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAzbeszt">
   <property name="text">
    <string>Azbeszt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">imageview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
